syntax="proto3";

package transport_proto;

import "svg.proto";

message RenderSettings {
	double width = 1;
	double height = 2;
	double padding = 3;
	double line_width = 4;
	double stop_radius = 5;
	int32 bus_label_font_size = 6;
	repeated double bus_label_offset = 7;
	int32 stop_label_font_size = 8;
	repeated double stop_label_offset = 9;
	Color underlayer_color = 10;
	double underlayer_width = 11;
	repeated Color color_palette = 12;
}